grind/python/disttest/isolate.py

```diff
@@ -97,14 +98,18 @@ def _generate_run_script_contents(self):
         exit $rc
     fi
 }
-export JAVA7_BUILD=true
-. /opt/toolchain/toolchain.sh
+
 run ./unpack.sh # Generated by packager
 source environment.source # Init runtime environment
+"""
+        contents += """
+jdks=(/usr/lib/jvm/java-{java_version}-*)
+export JAVA_HOME=${{jdks[0]}}
+""".format(java_version=self.java_version)
+
+        contents += """
 run which mvn
 run mvn -version
-run which java
```

Comment:
Is java not on the path at this point? I'd like to keep this, since it's nice for debugging (especially if we're going multi-JDK).